Source/Core/VideoCommon/TmemBase.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+
+namespace TmemBase
+{
+void Invalidate(u32 param);
+void ConfigurationChanged(int bp_addr, int config);
+void Bind(u8 unit, int num_blocks_width, int num_blocks_height, bool is_mipmapped, bool is_32_bit);
```

Comment:
These `u8`s should probably be `u32`. u32 is slightly more flexible when it comes to interface changes outside of this source file, given it's very easy to implicitly truncate to u8, causing warnings.